inference-engine/ie_bridges/java/org/intel/openvino/IECore.java

```diff
@@ -0,0 +1,95 @@
+package org.intel.openvino;
+
+import java.util.Map;
+
+public class IECore extends IEWrapper {
+    public static final String NATIVE_LIBRARY_NAME = "inference_engine_java_api";
+
+    public IECore() {
+        super(GetCore());
+    }
+
+    public IECore(String xmlConfigFile) {
+        super(GetCore_1(xmlConfigFile));
+    }
+
+    public CNNNetwork ReadNetwork(final String xml, final String bin) {
+        return new CNNNetwork(ReadNetwork1(nativeObj, xml, bin));
```

Comment:
In general case ReadNetwork also allows to read ONNX models. I propose to rename `xml` and `bin` to `modelPath`, `weightPath`.